### DISCUSSION TITLE
Krait Obelisk Shards

### DISCUSSION THREAD

**Discussion by madarel** [blish-hud/Blish-HUD#764] [Requested Marker Packs]
wondering if you could consider making trails and markers for the krait obelisk shards, i think there is 24? [https://wiki.guildwars2.com/wiki/Toxic_Krait_Historian](url) This is the wiki for them.

**Comment by Metallis** [comment]
[Toxic Krait Historian.zip](https://github.com/blish-hud/Blish-HUD/files/9626706/Toxic.Krait.Historian.zip)
Initial attempt, needs user testing before integrating into https://github.com/Metallis/Metal-Marker-Myriad

**Comment by Metallis** [comment]
This was completed and merged into MMM.
